<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>557</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Управляющее устройство</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Задание</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_task"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Комментарии</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_comments"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTape">
    <property name="title">
     <string>Лента</string>
    </property>
    <addaction name="actionSaveTape"/>
    <addaction name="actionLoadTape"/>
   </widget>
   <widget class="QMenu" name="menuExexute">
    <property name="title">
     <string>Выполнить</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStep"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuSpeed">
    <property name="title">
     <string>Скорость</string>
    </property>
    <addaction name="actionFast"/>
    <addaction name="actionNormal"/>
    <addaction name="actionSlow"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTape"/>
   <addaction name="menuExexute"/>
   <addaction name="menuSpeed"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Menu</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="actionRun"/>
   <addaction name="actionPause"/>
   <addaction name="actionStep"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Tape</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Alphabet</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="action_table_add_left"/>
   <addaction name="action_table_add_right"/>
   <addaction name="action_table_column_delete"/>
   <addaction name="actionRemoveCell"/>
  </widget>
  <action name="actionAbout_Programm">
   <property name="icon">
    <iconset>
     <normaloff>:/style/about.png</normaloff>:/style/about.png</iconset>
   </property>
   <property name="text">
    <string>Programm</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/new.png</normaloff>:/style/new.png</iconset>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/open.png</normaloff>:/style/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/save.png</normaloff>:/style/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/save.png</normaloff>:/style/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionLoadTape">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/import.png</normaloff>:/style/import.png</iconset>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/run.png</normaloff>:/style/run.png</iconset>
   </property>
   <property name="text">
    <string>Запуск</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/step.png</normaloff>:/style/step.png</iconset>
   </property>
   <property name="text">
    <string>Шаг</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/pause.png</normaloff>:/style/pause.png</iconset>
   </property>
   <property name="text">
    <string>Пауза</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="icon">
    <iconset>
     <normaloff>:/style/author.png</normaloff>:/style/author.png</iconset>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionSaveTape">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/export.png</normaloff>:/style/export.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionFast">
   <property name="text">
    <string>Быстро</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Средне</string>
   </property>
  </action>
  <action name="actionSlow">
   <property name="text">
    <string>Медленно</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/stop.png</normaloff>:/style/stop.png</iconset>
   </property>
   <property name="text">
    <string>Стоп</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/left.png</normaloff>:/style/left.png</iconset>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
   <property name="toolTip">
    <string>Prev</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/right.png</normaloff>:/style/right.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action_table_add_left">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/table_left.png</normaloff>:/style/table_left.png</iconset>
   </property>
   <property name="text">
    <string>Добавить столбец слева</string>
   </property>
   <property name="toolTip">
    <string>Добавить столбец слева</string>
   </property>
  </action>
  <action name="action_table_add_right">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/table_right.png</normaloff>:/style/table_right.png</iconset>
   </property>
   <property name="text">
    <string>Добавить столбец справа</string>
   </property>
   <property name="toolTip">
    <string>Добавить столбец справа</string>
   </property>
  </action>
  <action name="action_table_column_delete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/column_delete.png</normaloff>:/style/column_delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить столбец</string>
   </property>
   <property name="toolTip">
    <string>Удалить столбец</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/edit.png</normaloff>:/style/edit.png</iconset>
   </property>
   <property name="text">
    <string>Изменить алфавит</string>
   </property>
   <property name="toolTip">
    <string>Изменить алфавит</string>
   </property>
  </action>
  <action name="actionRemoveCell">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/cell_delete.png</normaloff>:/style/cell_delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить ячейку</string>
   </property>
   <property name="toolTip">
    <string>Удалить ячейку</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionAddTapeText">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/style/enter.png</normaloff>:/style/enter.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="toolTip">
    <string>Добавить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
